docs/windows.md

```diff
@@ -128,6 +128,33 @@ Install Gaphor's dependencies
 poetry install
 ```
 
+Reinstall PyGObject and pycairo using gvsbuild wheels
+```PowerShell
+poetry run pip install --force-reinstall (Resolve-Path C:\gtk\wheels\PyGObject*.whl)
+poetry run pip install --force-reinstall (Resolve-Path C:\gtk\wheels\pycairo*.whl)
+```
+
+Launch Gaphor!
+```PowerShell
+poetry run gaphor
+```
+
+### Setting Up A Plugin Workspace for Gaphor
+
+When setting up a plugin workspace you need to perform the following steps:
+
+cd (your project's workspace)
+
+If your project does not already have a pyproject.toml file, create one. For details see the [Poetry documentation](https://python-poetry.org/docs/basic-usage/)
+```PowerShell
+poetry init
+```
+
+Install your project's dependencies. If you have made your project dependent upon gaphor, this will pull in gaphor
```

Comment:
Note that Gaphor should be installed as _development_ dependency, like in the [hello-world plugin](https://github.com/gaphor/gaphor_plugin_helloworld/blob/main/pyproject.toml#L16-L18).

Reply:
Added a note to the line 148 instructions indicating that gaphor should be set up as a development dependency in the .toml file and referencing the hello world plugin. Won't show up until I check in the change.